src/App.tsx

```diff
@@ -15,71 +15,80 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const goodsList = goods.map(good => (
+    <tr
+      data-cy="Good"
+      key={good}
+      className={
+        (selectedGood === good)
+          ? ('has-background-success-light')
+          : ''
+      }
+    >
+      <td>
+        {
+          (selectedGood !== good)
+            ? (
+              <button
+                data-cy="AddButton"
+                type="button"
+                className="button"
+                onClick={() => setSelectedGood(good)}
+              >
+                +
+              </button>
+            )
+            : (
+              <button
+                data-cy="RemoveButton"
+                type="button"
+                className="button is-info"
+                onClick={() => setSelectedGood('')}
+              >
+                -
+              </button>
+            )
+        }
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+      </td>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+      <td data-cy="GoodTitle" className="is-vcentered">
+        {good}
+      </td>
+    </tr>
+  ));
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {
+          (!selectedGood
+            ? 'No goods selected'
+            : `${selectedGood} is selected`
+          )
+        }
 
-        <tr data-cy="Good">
-          <td>
+        {
+          selectedGood && (
```

Comment:
```suggestion
        {selectedGood && (
```

Reply:
Doesn't matter as the code written to be easily to read?
For me personally it's easier to scroll the code and find it, if I have it on separate lines, than to be "glued" like this.

Anyway, I made the changes as requested.

Thank you!

Reply:
to my mind this way we do not lose any clearance of the code, but it looks more ergonomical